pkgs/applications/video/vdr/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv, fetchurl, fontconfig, libjpeg, libcap, freetype, fribidi, pkgconfig
+, gettext, ncurses, systemd, perl
+, enableSystemd ? true
+, enableBidi ? true
+}:
+let
+
+  version = "2.4.0";
+
+  name = "vdr-${version}";
+
+  mkPlugin = name: stdenv.mkDerivation {
+    name = "vdr-${name}-${version}";
+    inherit (vdr) src;
+    buildInputs = [ gettext vdr ];
+    nativeBuildInputs = [ pkgconfig ];
+    preConfigure = "cd PLUGINS/src/${name}";
+    installFlags = [ "DESTDIR=$(out)" "PREFIX=" ];
```

Comment:
Usually this comes with its own set of problems using DESTDIR instead of PREFIX because it uses PREFIX somewhere in the source. What was the offending file that could that ignored PREFIX, when installing?